typescript/rest-nuxtjs/server/routes/publish/[id].put.ts

```diff
@@ -0,0 +1,20 @@
+import { prisma } from '../../../prisma/db'
+
+// https://nuxt.com/docs/guide/directory-structure/server
+export default defineEventHandler(async (event) => {
+    const { context: { params: { id } } } = event;
```

Comment:
```suggestion
    const id = event.context.params;
```

Reply:
Ok so I think there was a mistake with the change request should:
`const id = event.context.params.id;`
So I assume thats what you meant. So I used that instead. Its Done.